src/browser/components/Tab/Tab.scss

```diff
@@ -1,3 +1,10 @@
+// Tab Component Colors
+$tab-active-color: #f8f8f8;
```

Comment:
@tjuretzek looks better! would suggest the following:

* remove the comments, they seem a bit redundant
* be specific about the property we intend, so color -> background-color
* consider whether the value is likely to be reused and whether we need the variable at all
* remove the default keyword, and just attach the property directly to the component name
* list component properties first, then states, then sub-components

```
$tab-height: 38px;
$tab-background-color: #eee
$tab-active-background-color: #f8f8f8
```

Reply:
done and done